src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -34,8 +34,10 @@ class SeekBar extends Slider {
    */
   constructor(player, options) {
     super(player, options);
-    this.update = Fn.throttle(Fn.bind(this, this.update), 50);
-    this.on(player, ['timeupdate', 'ended'], this.update);
+    this.throttleUpdate = Fn.throttle(Fn.bind(this, this.update), 50);
```

Comment:
I'd be inclined to keep this `this.update`. If people call the `update` method they won't be throttled with this change which could be unexpected.

Reply:
Would you be okay with adding a `@deprecated` comment here so that it can be changed in a major version bump? I don't think we want to modify functions directly, so that they can be called without a throttle.

Reply:
internally, with your change, `update_` is available to be called unthrottled, no?

Reply:
true, yeah lest leave it at that